src/stepfunctions/steps/service.py

```diff
@@ -32,6 +33,14 @@ class DynamoDBApi(Enum):
     DeleteItem = "deleteItem"
     UpdateItem = "updateItem"
 
+class EKSApi(Enum):
+    CreateCluster = "createCluster"
+    DeleteCluster = "deleteCluster"
+    CreateFargateProfile = "createFargateProfile"
+    DeleteFargateProfile = "deleteFargateProfile"
+    CreateNodegroup = "createNodegroup"
+    DeleteNodegroup = "deleteNodegroup"
```

Comment:
how about the `eks:runJob` and `eks:call` integrations mentioned in the [documentation](https://docs.aws.amazon.com/step-functions/latest/dg/connect-eks.html#connect-eks-kubernetes-apis)

is there a reason we are omitting those?

Reply:
I had omitted them since they were grouped under `Kubernetes API integrations`

As discussed offline, it makes sense to include them in this PR as they are part of the same integration and follow the same naming convention. Adding them in the next commit